vocab_files/observable_property_concepts/mean-basal-area.ttl

```diff
@@ -6,7 +6,7 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 <https://linked.data.gov.au/def/nrm/a70282a3-c62f-43df-8d9c-34727ae1ee16>
     a skos:Concept ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
-    skos:definition "TBD" ;
+    skos:definition "The average basal area for a given plot, expressed in square meters (m^2) per unit area of the plot." ;
```

Comment:
the definition should be "The average amount of area for a given plot occupied by tree stems, expressed in square meters (m^2) per unit area of the plot." ; I am not sure whether this is correct,  I don't think they do average; it is just a total cross-sectional area of all stems  measured and expressed as sq mt/hectare. probably good to confirm

Reply:
probably it's added in the app and dataset, from the protocol:
_Repeat steps 3–11 at each remaining sampling location. When all the sampling locations have been assessed, the App will display a data summary (e.g. mean basal area, total stem counts) on the sampling location selection screen._